### ISSUE TITLE
Dataloader default value for shuffle

### ISSUE THREAD

**Issue by dhorka** [rusty1s/pytorch_geometric#322]
I noticed that the data loader has as default value for shuffle true, however, the default value for shuffle in pytorch dataloader is false. I would suggest to follow the same logic as pytorch library and put the default value to false.

**Comment by rusty1s** [comment]
Fixed in master. Thank you.
